ast-nodes/variable-declaration.dart

```diff
@@ -79,7 +79,12 @@ class VariableDeclaration extends Declaration {
   }
 
   Pointer<LLVMOpaqueValue> generateCode(Module module) {
-    // TODO: implement
+    this.valueRef = llvm.LLVMBuildAlloca(module.builder,
```

Comment:
Also, since you're going for level 6, don't forget that every statement should be put in its own basic block, otherwise it will segfault